### ISSUE TITLE
Modal focus trap

### ISSUE THREAD

**Issue by amnesca** [robsontenorio/mary#767]
### maryUI version

Version used on doc site

### daisyUI version

Version used on doc site

### Livewire version

Version used on doc site

### What browsers are affected?

Firefox, Chrome

### What happened?

I just noticed that the [Modal component](https://mary-ui.com/docs/components/modal) on the doc site doesn't have  focus trap applied, as continued tabbing starts to focus on items in the background page behind the open modal. This doesn't seem to happen on the Drawer component though. Not sure how it's set up, but perhaps the Alpine focus trap hasn't been configured or should be applied?

**Comment by robsontenorio** [comment]
Yes, probably the solution is to use same approach of drawers. There is a alpine plugin to do this.

**Comment by amnesca** [comment]
Thanks Robson, I just used the updated modal component in a project and it works great. Cheers.
